src/types.ts

```diff
@@ -593,7 +593,9 @@ export type SearchWarning = {
   warning_code: number;
   warning_description: string;
 };
-export type SendFileAPIResponse = APIResponse & { file: string };
+
+// Thumb URL(thumb_url) is added considering video attachments as the backend will return the thumbnail in the response.
+export type SendFileAPIResponse = APIResponse & { file: string; thumb_url: string };
```

Comment:
thumb_url should be optional

Reply:
Yes, my mistake, already pushed with the changes. 😄